x-pack/plugins/infra/public/pages/metrics/hosts/components/kpis/kpi_grid.tsx

```diff
@@ -54,15 +52,15 @@ const KPI_CHARTS: Array<Omit<KPIChartProps, 'loading' | 'subtitle'>> = [
     title: i18n.translate('xpack.infra.hostsViewPage.metricTrend.diskSpaceUsage.title', {
       defaultMessage: 'Disk Space Usage',
     }),
-    toolTip: i18n.translate('xpack.infra.hostsViewPage.metricTrend.diskSpaceUsage.tooltip', {
-      defaultMessage: 'Percentage of disk space used.',
-    }),
+    toolTip: TOOLTIP.tx,
   },
 ];
 
 export const KPIGrid = () => {
   return (
     <HostCountProvider>
+      <HostMetricsDocsLink />
```

Comment:
This is a valid point. The idea is to provide easy access to the documentation. @roshan-elastic , do you believe adding a link here will add more value?

Reply:
Hey @mohamedhamed-ahmed - this was added only because the black tooltips aren't able to show links to the docs:

![image](https://github.com/elastic/kibana/assets/117740680/90ba365f-ca4e-4a4e-a1d9-f046448a0c66)

The white tooltips below show links to the docs but I wasn't sure if users would find these so put this as a fallback:

![image](https://github.com/elastic/kibana/assets/117740680/ae400c02-87d5-4441-983d-3efc8ffc3133)

I'm asking one of our SREs to quickly get a sense. If I don't hear back, let's keep this in (we can always remove later if we see nobody clicks on it/users don't like it).

Reply:
No worries @mohamedhamed-ahmed - understand the ask...it's a good Q.

Did a quick sense check with the SRE and he preferred it so let's keep it